packages/web-components/src/tabs/tab-list/tab-list.ts

```diff
@@ -0,0 +1,82 @@
+import { attr, booleanConverter } from '@microsoft/fast-element';
+import { FASTTabs } from '@microsoft/fast-foundation';
+
+export interface TabData {
+  id: string;
+  x: number;
+  y: number;
+  height: number;
+  width: number;
+}
+/**
+ * TabList extends FASTTabs and is used for constructing a fluent-tab-list custom html element.
+ *
+ * @class TabList component
+ * @public
+ */
+export class TabList extends FASTTabs {
+  @attr appearance?: 'subtle' | 'transparent';
+
+  @attr({ converter: booleanConverter })
+  disabled?: boolean;
+
+  @attr size?: 'small' | 'medium' | 'large' = 'medium';
+
+  @attr({ attribute: 'reserve-selected-tab-space', converter: booleanConverter })
+  reserveSelectedTabSpace?: boolean;
+
+  activeidChanged(oldValue: string, newValue: string) {
+    super.activeidChanged(oldValue, newValue);
+    this.setTabData();
+  }
+
+  tabsChanged(): void {
+    super.tabsChanged();
+    this.setTabData();
+  }
+
+  private setTabData(): void {
+    if (this.tabs && this.tabs.length > 0) {
+      const activeTab = this.tabs.filter(tab => tab.id === this.activeid)[0] || this.tabs[0];
+      const activeRect = activeTab?.getBoundingClientRect();
+      const parentRect = this.getBoundingClientRect();
+
+      this.tabs.forEach(tab => {
+        tab.dataset.activeTab = JSON.stringify({
+          id: activeTab.id,
+          x: activeRect.x - parentRect.x,
+          y: activeRect.y - parentRect.y,
+          height: activeRect.height,
+          width: activeRect.width,
+        });
+      });
+    }
+  }
+
+  // Remove this method after updates to Fast Tabs Control are completed
+  disabledChanged() {
+    const tabs = this.querySelectorAll('fluent-tab');
+
+    if (this.disabled === true) {
+      tabs.forEach(function (tab, index) {
+        tab.setAttribute('disabled', 'true');
+      });
+
+      /* 
+      after setting all tabs to disabled the FAST tabs control 
+      sets showActiveIndicator to false. Fluent guidelines have the tab indicator always showing if the tab is active.
```

Comment:
I think we discussed this offline, but our current hope/approach is to remove the prpescriptive `activeIndiciator` and all of the JS because it hoists presentational details to the whole component. Were we not able to do this without using that element? I believe you mentioned that the actual current v9 implementation uses a pseudo + animation.